src/sweeper/__main__.py

```diff
@@ -63,15 +74,27 @@ def main():
         closet.append(EmptyTest(args['--workspace'], args['--table-name']))
         closet.append(MetadataTest(args['--workspace'], args['--table-name']))
 
-    reports = execute_sweepers(closet, args['--try-fix'])
+    reports = execute_sweepers(closet, args['--try-fix'], args['--change-detect'])
 
     report.print_report(reports)
 
     if args['--save-report']:
         report.save_report(reports, args['--save-report'])
 
+    if args['--scheduled']:
+        final_message = report.format_message(reports)
+        print(final_message.getvalue())
+        
+        #: Build and send summary message
+        summary_message = MessageDetails()
+        summary_message.message = final_message.getvalue()
+        summary_message.project_name = 'agrc-sweeper'
+        summary_message.subject = f'Sweeper Report {datetime.datetime.today()}'
+
+        sweeper_supervisor.notify(summary_message)
+
 
-def execute_sweepers(closet, try_fix):
+def execute_sweepers(closet, try_fix, change_detect):
```

Comment:
**suggestion**: this variable name might be more descriptive if it was something like, `using_change_detection`

Reply:
I'll update it.